backend/vaa-strapi/src/functions/generateMockData.ts

```diff
@@ -507,173 +508,11 @@ async function createQuestionCategories() {
 }
 
 async function createQuestionTypes() {
-  const questionTypes: {
+  for (const questionType of mockQuestionTypes as Array<{
```

Comment:
Moved most mock data sources to their own json files so that we can import them into tests. In the future, it might behoove us to move them to a shared module (and use a the same format for the data as can be imported directly with `LocalDataProvider`.

Reply:
I think so, although currently the mock data is only applicable to the app-wide e2e tests in `<root>/tests` (and there are no backend unit tests). However, when the `DataProvider` update is complete, we should use the same data to run tests with `LocalDataProvider`, which is part of the frontend, and also for development instances.

I added this to issue #148